smart-contract/assembly/__tests__/deweb-interface.spec.ts

```diff
@@ -107,14 +121,133 @@ describe('website deployer internals functions tests', () => {
     });
 
     throws('should throw if file does not exist', () => {
-      getChunk(chunkGetArgs(file1NameHash, 0));
+      getChunk(chunkGetArgs(file1PathHash, 0));
     });
   });
-});
 
-function chunkGetArgs(
-  filePathHash: StaticArray<u8>,
-  index: u32,
-): StaticArray<u8> {
-  return new ChunkGet(filePathHash, index).serialize();
-}
+  // Testing delete files
+  describe('Delete File', () => {
+    beforeEach(() => {
+      resetStorage();
+      setDeployContext(user);
+      constructor(new Args().serialize());
+    });
+
+    test('Should delete 1 file with 1 chunk', () => {
```

Comment:
I have to improve this myself but we should make the test title coherent with the function used. In your case we can use `it` insted of `test` or change the title by `test('that we can delete 1 file with 1 chunk')`